src/styles.css

```diff
@@ -17,4 +17,10 @@ body {
   justify-content: center;
   align-items: center;
   border-top: 1px solid black;
+}
+
+.content {
+  min-height: calc(100vh - 100px);
+  width: 720px;
```

Comment:
Set an addition  `max-width` here to make the container responsive and you should be done